src/views/EditSidebar.vue

```diff
@@ -201,17 +206,24 @@
 				@save-this-and-all-future="saveAndLeave(true)" />
 		</AppSidebarTab>
 		<AppSidebarTab v-if="!isLoading && !isError"
-			id="app-sidebar-tab-resources"
+			id="app-sidebar-tab-media"
 			class="app-sidebar-tab"
-			:name="$t('calendar', 'Resources')"
+			:name="$t('calendar', 'Media')"
 			:order="2">
 			<template #icon>
-				<MapMarker :size="20" decorative />
+				<FolderMultipleImage :size="20" decorative />
 			</template>
 			<div class="app-sidebar-tab__content">
-				<ResourceList v-if="!isLoading"
-					:calendar-object-instance="calendarObjectInstance"
-					:is-read-only="isReadOnly" />
+				<!-- FIXME provide integer calendar id -->
+				<RelatedResourcesPanel provider-id="calendar"
+					:item-id="calendarId"
```

Comment:
@st3iny @ChristophWurst related resources is implemented in a way that uses the integer calendar id https://github.com/nextcloud/related_resources/blob/31f3b2864d2a176686a25bda85d43e6c77b2c082/lib/RelatedResourceProviders/CalendarRelatedResourceProvider.php#L137, from @ArtificialOwl

The `calendarId` in this context is an alphanumeric string. How do we get the integer id here?

Reply:
Any insights @st3iny?

Reply:
@ChristophWurst can you help ?

Reply:
(you need both the calendar URI and the principal URI to identify the calendar)

Reply:
In that case it sounds like the backend needs to be updated to use URI's instead of the integer id

Is this possible @ArtificialOwl?

Reply:
sure, send me an example of your data, i'll accommodate the backend

Reply:
Commits added to use the item id format: `{principalUri}:{calendarUri}`